plat/common/aarch64/plat_common.c

```diff
@@ -56,11 +56,13 @@ void bl32_plat_enable_mmu(uint32_t flags)
 
 void bl31_plat_runtime_setup(void)
 {
+#ifndef DEBUG
```

Comment:
You just need to force-push your local branch again. I.e. just rerun your original push command with a -f. Github will update the PR automatically (as long as the remote branch is the same, I think).